src/InstrumentationEngine.Preinstall/InstrumentationEngine.Preinstall.csproj

```diff
@@ -85,4 +85,11 @@
       <Output TaskParameter="DestinationFiles" ItemName="FileWrites" />
     </Copy>
   </Target>
+  <Target Name="ProducePatchedXdt">
+    <PropertyGroup>
+      <Xdt_DisableSignatureValidation Condition=" '$(DisableSignatureValidation)'=='' ">$False</Xdt_DisableSignatureValidation>
```

Comment:
>Xdt_DisableSignatureValidation [](start = 7, length = 30)

Either the condition needs to be Xdt_DisableSignatureValidation or you need a secondary statement or choose to consume $(DisableSignatureValidation). Also who would set $DisableSignatureValidation? If you expect the build machine to control this, consider using upper casing, since it will come from an environment variable.

Reply:
I envisioned these as build parameters, will change to upper casing

---
In reply to: [322461511](https://github.com/microsoft/CLRInstrumentationEngine/pull/133#discussion_r322461511) [](ancestors = 322461511)